plugins/@grouparoo/spec-helper/src/lib/factories/properties.ts

```diff
@@ -15,7 +17,14 @@ export default async (
     purchaseAmounts: "float",
   }
 ) => {
-  const source = (await SourceFactory()) as Source;
+  if (!modelId) modelId = "mod_profiles";
+  const model =
+    (modelId
+      ? await GrouparooModel.findOne({ where: { id: modelId } })
+      : await GrouparooModel.findOne()) ??
+    ((await ModelFactory({ id: modelId })) as GrouparooModel);
```

Comment:
The goal of these factories is to set up everything you need for you, so if you ask for "properties", now we will make you a model, app, and source (previously just app and source).  You can also pass a modelId in now to use an existing model if you want.

most of our tests setup the world by calling `helper.factories.properties()` so this seems the most helpful.